src/Tabs/InkBar.js

```diff
@@ -16,6 +16,7 @@ function getStyles(props, context) {
       marginTop: -2,
       position: 'relative',
       transition: transitions.easeOut('1s', isRtl ? 'right' : 'left'),
+      zIndex: 1000,
```

Comment:
Where is this value coming from? Is it relative to one of [those](https://github.com/callemall/material-ui/blob/master/src/styles/zIndex.js)?

Reply:
@oliviertassinari making zIndex: 1 also solves the problem, When we pass custom style to Tab, InkBar disappears not sure what is causing the issue but adding zIndex: 1 in Tabs/InkBar.js root style solves the problem. 

Code to  reproduce the issue:

```javascript
  <Tabs
        value={this.state.value}
        onChange={this.handleChange}
      >
        <Tab label="Upload Notes" value="Upload Notes"  buttonStyle={{backgroundColor: '#4d86cf'}} icon={<FileFileUpload />}>
          <div>
            <NotesUpload userrole={this.props.userrole} branch={this.props.branch} />
          </div>
        </Tab>
        <Tab label="View Notes" value="View Notes" buttonStyle={{backgroundColor: '#4d86cf'}} icon={<ActionViewList />}>
          <NotesList  userrole={this.props.userrole} branch={this.props.branch} />
        </Tab>
      </Tabs>
```
This code doesn't show InkBar under the tabs when buttonStyle parameter is passed and removing that parameter works perfectly fine.

Please reply to this thread if you need any more info on this.

Reply:
@oliviertassinari  any updates on this?